tests/__init.test.js

```diff
@@ -5,6 +5,24 @@
  */
 window.debug = true;
 
+// getVRDisplays mock
+navigator.getVRDisplays = function () {
```

Comment:
Can you expand on this? How is the code you propose equivalent? If I want to mock ```getVRDisplays``` I need to return a promise right? Your function returns undefined.